package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aframe-core",
-  "version": "0.1.3",
+  "version": "0.1.4",
```

Comment:
I know you know, but just a reminder: going to have to bump [`aframe-core` in `aframe`'s `package.json`](https://github.com/aframevr/aframe/blob/master/package.json#L35)